test/videoroom_web/controllers/room_json_test.exs

```diff
@@ -177,4 +213,15 @@ defmodule VideoroomWeb.RoomJsonTest do
     unless async?,
       do: assert_receive({:jellyfish, %PeerDisconnected{}}, @timeout)
   end
+
+  defp await_meeting_restart(name, prev_pid) do
+    case Registry.lookup(Videoroom.Registry, name) do
+      [{pid, _key}] when pid != prev_pid ->
+        :ok
+
+      _other ->
+        Process.sleep(10)
```

Comment:
Instead of sleep, you could define a listener for registry, which seems more elegant, but may not be that easy to implement - if you think it's not worth it, I'd keep that sleep

Reply:
I expect the Jellyfish to be empty, since upon creation the new peer is added to the local state and if a peer leaves the room, the local state is changed later. This means, that sometimes we can still have a peer in the local state but not in Jellyfish, but not the other way around.
I added error log in case this isn't true.

Reply:
OK, so one more thing - I'd limit the number of iterations here - the test should fail, not loop infinitely if something goes wrong.